lib/KImg.vue

```diff
@@ -115,6 +124,9 @@
       imgMinWidth() {
         return this.validateAndFormatUnits(this.minWidth);
       },
+      logo() {
```

Comment:
Since the item being shown won't always be a "logo" maybe we can call this something like `imgSrc`